### ISSUE TITLE
`ecs-cli configure --cluster C`  erases all the other settings

### ISSUE THREAD

**Issue by markriggins** [aws/amazon-ecs-cli#176]
Is this intentional?    Should `ecs-cli configure --cluster C` erase all the other configuration values?

**How to reproduce this problem:**
create a default config file with a cluster and region
```
$ echo '' >home/.ecs/config
$ HOME=$PWD/home ecs-cli.0.4.4 configure --cluster a-really-long-long-name --region us-east-1
$ cat home/.ecs/config
[ecs]
cluster                     = a-really-long-long-name
aws_profile                 = 
region                      = us-east-1
aws_access_key_id           = 
aws_secret_access_key       = 
compose-project-name-prefix = ecscompose-
compose-service-name-prefix = ecscompose-service-
cfn-stack-name-prefix       = amazon-ecs-cli-setup-
```

Now try to change the cluster:
```
$ HOME=$PWD/home ecs-cli.0.4.4 configure --cluster a-really-longer-longer-longer-longer-name
$ cat home/.ecs/config
[ecs]
cluster                     = a-really-longer-longer-longer-longer-name
aws_profile                 = 
region                      = 
aws_access_key_id           = 
aws_secret_access_key       = 
compose-project-name-prefix = ecscompose-
compose-service-name-prefix = ecscompose-service-
cfn-stack-name-prefix       = amazon-ecs-cli-setup-
```

And we see that it has forgotten the **region**.   

Do we really have to give you **EVERY ARGUMENT** every time we use the `ecs-cli configure` command?

**Comment by yinshiua** [comment]
Added proposals to support multiple clusters configuration. Please give us some feedback. Thanks! https://github.com/aws/amazon-ecs-cli/issues/253

**Comment by yinshiua** [comment]
Released in v1.0.0
